motor/frameworks/asyncio/__init__.py

```diff
@@ -16,6 +16,7 @@
 
 See "Frameworks" in the Developer Guide.
 """
+
```

Comment:
A lot of the changes in this PR seem to be formatting/typing changes unrelated to pymongo. Can you apply these in a separate PR to reduce the noise?

Reply:
Done

Reply:
Thanks!